.circleci/config.yml

```diff
@@ -59,7 +59,7 @@ jobs:
     environment:
       SETUPTOOLS_USE_DISTUTILS: stdlib
     machine:
-      image: ubuntu-2204:2023.07.2
```

Comment:
Is this intentionally part of this commit? (Not sure if it makes a difference one way or another, but curious about it)

Reply:
https://discuss.circleci.com/t/linux-image-deprecations-and-eol-for-2024/50177

Reply:
Ah, makes sense!